ABMobility/Grid.pde

```diff
@@ -31,11 +32,25 @@ public class Grid {
      
      table = loadTable("block/Cooper Hewitt Buildings - Building Blocks.csv", "header");
      for (TableRow row : table.rows()) {
-       buildings.add(new Building(gridMap.get(row.getInt("id")),row.getInt("id"),row.getInt("R"),row.getInt("O"),row.getInt("A")));
```

Comment:
@agrignard at this line the buildings were being instantiated, but they were not then being used.